azure-pipelines.yml

```diff
@@ -0,0 +1,69 @@
+trigger:
+  branches:
+    include:
+      - cccs-*
+      - feature/CLDN-*
```

Comment:
In general, we don't include feature branches in the trigger. All PRs will be automatically built, so when devs are ready to build they can create a PR.